### ISSUE TITLE
Affected by CVE-2022-37454

### ISSUE THREAD

**Issue by stefanor** [tiran/pysha3#29]
Clearly this backport module is unmaintained these days. But there are still things using it (like https://github.com/opentimestamps/python-opentimestamps).

See: https://github.com/python/cpython/pull/98527

**Comment by tiran** [comment]
Yes, this package is affected by the buffer overflow. However I'm retiring the project. Any project should use SHA-3 from Python's hashlib module instead.
